package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "webdb-iii-challenge",
+  "version": "1.0.0",
+  "description": "## Topics",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "server": "nodemon index.js",
+    "start": "node index.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/MaxGunter99/webdb-iii-challenge.git"
+  },
+  "keywords": [
+    "knex",
+    "express",
+    "api",
+    "database"
+  ],
+  "author": "Michael Gunter",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/MaxGunter99/webdb-iii-challenge/issues"
+  },
+  "homepage": "https://github.com/MaxGunter99/webdb-iii-challenge#readme",
+  "dependencies": {
+    "cors": "^2.8.5",
+    "express": "^4.16.4",
+    "helmet": "^3.18.0",
+    "knex": "^0.16.5",
+    "morgan": "^1.9.1",
+    "sqlite3": "^4.0.8"
+  }
```

Comment:
dont forget to add nodemon as a dev dependency of the project if you have a script using it for future devs that may use te project and not have it globally installed  in their system